template/skaffold.yaml.jinja

```diff
@@ -11,36 +11,70 @@ build:
             path: Dockerfile
 deploy:
   helm:
+    hooks:
+      before:
+        - host:
+            command:
+              - sh
+              - -c
+              - helm repo add bitnami https://charts.bitnami.com/bitnami && helm repo add tnlcommunity https://raw.githubusercontent.com/tnlcommunity/charts/main && helm repo add catalystsquad https://raw.githubusercontent.com/catalystsquad/charts/main
```

Comment:
could remove the tnlcommunity chart, since there's no project there that's used in this hello app

Reply:
maybe the chart repos need to be conditional, the bitnami repo has postgres, but the cockroach chart's repo isn't in here, so if it's selected in the copier options then the repo won't be installed. this is really low on the list because i don't foresee us using cockroach very much